evansmusomi/burger-builder
# Fetches Orders through Redux Actions

*What this does*
- Fetches orders from Firebase via Redux actions

*How to verify*
- A user can view orders stored on the server when they navigate to the orders page
- A developer can confirm that the redux store and actions are used using dev tools
